src/pages/Mathematics.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import Calculator from '../components/Calculator.js';
```

Comment:
- [ ] The reason for the error above is you are importing components/Calculator.js but you should be importing components/calculator.js.

Reply:
But I don't have calculator.js.
I instead have Calculator.js. And it runs fine on my side

Reply:
Fixed

Reply:
Good Job

![fullsite](https://user-images.githubusercontent.com/55635977/144391390-62601713-a4fa-47f6-890d-b775a4e5c66c.png)